cli/cli.hs

```diff
@@ -144,6 +144,7 @@ data Command w
       , smttimeout    :: w ::: Maybe Natural            <?> "Timeout given to SMT solver in seconds (default: 300)"
       , maxIterations :: w ::: Maybe Integer            <?> "Number of times we may revisit a particular branching point"
       , loopDetectionHeuristic :: w ::: LoopHeuristic   <!> "StackBased" <?> "Which heuristic should be used to determine if we are in a loop: StackBased (default) or Naive"
+      , abstRefine    :: w ::: Bool                      <?> "Use abstraction refinement"
```

Comment:
I think the name could be clearer here. I'm also wondering if we want to be more granular in the options that we expose? Perhaps at least a separate option for memory and the arithmetic ops?

Reply:
Do we wanna default to True or False here btw?

Reply:
I would be  fine with switching to cvc5 as the default, but agree that it can be done in a separate pr.